service.go

```diff
@@ -52,11 +68,20 @@ func (s *topicServiceImpl) getTopicByUUID(uuid string) (topic, bool) {
 	return topic, found
 }
 
-func (s *topicServiceImpl) initTopicsMap(terms []term) {
-	for _, t := range terms {
-		top := transformTopic(t)
+func (s *topicServiceImpl) checkConnectivity() error {
+	// TODO: Can we just hit an endpoint to check if TME is available? Or do we need to make sure we get genre taxonmies back? Maybe a healthcheck or gtg endpoint?
+	_, err := s.repository.GetTmeTermsFromIndex(1)
```

Comment:
We looked into this and we couldn't see an immediate balance between a lack of ping/healthcheck on the TME side and the load that we would put TME for those calls. I agree it needs a chat as it will affect the healthcheck solution for all the V1 rw apps